### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by KonradIT**
- max resp len seems to be 500
- added score in hashtags

### CODE REVIEW

File: tests/test_hashtags.py [hunk 1]
```diff
@@ -12,4 +12,4 @@ def test_suggested_hashtags():
 
     # Later results (e.g., past #5) won't have expanded info
     assert "hashtag" in resp[0]
-    assert "description" in resp[0]
+    assert "score" in resp[0]
```
Comment (milesmcc) [inline tests/test_hashtags.py]: I'm not currently getting `score` in `resp[0]`, so I'll just change this to be:

```python
    assert ("description" in resp[0]) or ("score" in resp[0])
```

File: tests/test_search.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 def test_basic_search():
     """Verifies we can perform a basic search."""
     posts = list(client.search(query="America", max=1776))
-    assert len(posts) == 1776
+    assert len(posts) == 500
```
Comment (milesmcc) [inline tests/test_search.py]: Sometimes I'm getting 499 results. I'll change this to verify it's > 400.

File: tests/test_search.py [hunk 3]
```diff
@@ -20,4 +20,4 @@ def test_basic_search():
 def test_paginated_search():
     """Verifies we can perform a paginated search."""
     posts = list(client.search(query="x", max=5000))
-    assert len(posts) == 5000
+    assert len(posts) == 500
```
Comment (milesmcc) [inline tests/test_search.py]: Same as above.

**Review by milesmcc — APPROVED** [review]

**Comment by milesmcc** [comment]
Thanks so much!

**Comment by KonradIT** [comment]
Ah perfect, much better this way, weird how gettr keeps changing the max length of array response.

**Pull request merged**